atlasapi/atlas.py

```diff
@@ -410,8 +410,12 @@ def pause_cluster(self, cluster: str, toggle_if_paused: bool = False) -> dict:
             elif existing_config.paused is True and toggle_if_paused is True:
                 logger.warning('Cluster is paused, will toggle to unpaused, since toggle_if paused is true')
                 new_config = dict(paused=False)
-            else:
+            elif toggle_if_paused is False:
                 new_config = dict(paused=True)
+            else:
+                logger.error("The cluster is unpaused. No need to unpause again.")
+                raise ErrAtlasBadRequest(400,
```

Comment:
I feel there is no need to raise an exception, but I am not sure what to return.
MGM?